src/components/ActionButtonGroup/ActionButtonGroup.tsx

```diff
@@ -68,6 +72,11 @@ const ActionButtonGroup = ({ children }: ActionButtonGroupProps) => {
     return rowData.slice(1)
   }, [rowData]);
 
+  const handleClick = (e: any) => {
```

Comment:
nit: can we call this something like `handleClickShowMore`?

Reply:
also I think the arg should be optional here

Reply:
good call!